archwiki-rs: add page (#12700)
tldr-pages/tldr: pages/common/archwiki-rs.md

# archwiki-rs

> > A CLI tool to read pages from the ArchWiki.

Reviewer: ```suggestion
> Read, search, and download ArchWiki pages.
```
We generally avoid descriptions with CLI/command-line and prefer starting with a verb in the imperative mood, as document in contributing-guides/style-guide.md.

What about this?

> More information: <https://gitlab.com/lucifayr/archwiki-rs>.

- Read a page from the ArchWiki:

`archwiki-rs read-page {{page-title}}`

- Read a page from the ArchWiki in the specified format:

> `archwiki-rs read-page {{page-title}} --format {{plain-text|markdown|html}}`

Reviewer: ```suggestion
`archwiki-rs read-page {{page_title}} --format {{plain-text|markdown|html}}`
```
We use _snake_case_ in placeholders

> - Search the ArchWiki for pages containing the text "/usr/share":
> 
> `archwiki-rs search "/usr/share" --text-search`

Reviewer: We can use a placeholder here.
```suggestion
- Search the ArchWiki for pages containing a specific text:

`archwiki-rs search "{{search_text}}" --text-search`
```
Or:
```suggestion
- Search the ArchWiki for pages containing a specific text:

`archwiki-rs search "{{/usr/share}}" --text-search`
```

> - Download a local copy of all ArchWiki pages:
> 
> `archwiki-rs local-wiki ~/archwiki-html --format html`

Reviewer: ```suggestion
- Download a local copy of all ArchWiki pages in a specific directory:

`archwiki-rs local-wiki {{path/to/download_directory}}  --format {{{plain-text|markdown|html}}`
```
We can use these two placeholders here. The second one doesn't need an explanation because it already has one in a previous example.
